ui/src/shared/components/TableGraph.js

```diff
@@ -233,6 +233,9 @@ class TableGraph extends Component {
 
     const cellData = processedData[rowIndex][columnIndex]
 
+    const timeFieldIndex = fieldNames.findIndex(
```

Comment:
consider merging the logic of finding timeFieldIndex and timeField since you are essentially doing the same thing twice.